template/config/@control.ts/min/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@control.ts/min": "^0.1.10"
```

Comment:
Can we use `latest`?

Reply:
![image](https://github.com/TELEUZI/create-control.ts/assets/81071714/72d17ba7-6c59-43f2-937a-f8e174daafb4)